src/components/Signup/index.tsx

```diff
@@ -1,30 +1,95 @@
+import './styles/Signup.css'
 export interface Props {
     signUpAuthenication: Function;
 }
- 
+
 export interface State {
     username: string;
     email?: string;
     password: string;
-    correctPassword: string;
+    verifiedPassword: string;
+    error: boolean;
 }
- 
+
 class Signup extends React.Component<Props, State> {
     constructor(props: Props) {
         super(props);
-        this.state = { 
-            username: 'name',
-            email: "@gmail.com",
-            password: 'password',
-            correctPassword: 'Same Password'
+        this.state = {
+            username: '',
+            email: "",
+            password: '',
+            verifiedPassword: '',
+            error: false
         };
     }
-    render() { 
-        return ( <div>
-            <h1>hello</h1>
-        </div> );
+
+    handleSignUp = () =>{
+        //#TODO update url to meet api standard
+
+        fetch(`https://calcumon-user-api.herokuapp.com/user/`,{
+          method: 'post',
+            headers: {
+              'accept': 'application/json',
+              'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({email: this.state.email, username: this.state.username, password: this.state.password})
+          })
+          .then((response)=>{
+            console.log(response)
+            this.setState({})
+          })
+          .catch((err)=>{
+            console.log(err)
+          })
+      }
+
+    render() {
+        return (
+        <div>
+            <div className="singUpContainer">
+            <input type="password"
+                    name="inputPassword"
+                    id="inputPassword"
+                    className="form-control"
+                    placeholder="email"
+                    onChange={(e) => this.setState({ email : e.target.value})}
+                    required />
+              <input type="string"
+                    name="username"
+                    id="inputUsername"
+                    placeholder="Username"
+                    onChange={(e) => this.setState({ username : e.target.value})}
+                    required autoFocus />
+              <input type="password"
+                    name="inputPassword"
+                    id="inputPassword"
+                    className="form-control"
+                    placeholder="Password"
+                    onChange={(e) => this.setState({ password : e.target.value})}
+                    required />
+              {/* <input type="password"
+                    name="inputPassword"
+                    id="inputPassword"
+                    className="form-control"
+                    placeholder="Verify Password"
+                    onChange={(e) => this.setState({ verifiedPassword : e.target.value})}
+                    required /> */}
+              <button id="submitButton" type="submit" onClick={() => {
+                // TODO: validate data function
```

Comment:
An example of validate data functions I wrote if it helps --> https://github.com/Anisha7/sadboykj-site/blob/2e7e83b733142a0d998e0726d323ff732cb9bb8b/client/src/helpers/InputValidation/index.js
You could copy paste some functions and modify it. The only tricky one would be validating password, since we first need to come up with what requirements we have for password and show the users those requirements.